### ISSUE TITLE
Deleting a note, and then cancelling, crashes the app

### ISSUE THREAD

**Issue by azureslumbag** [rajatkulkarni95/octarine-feedback#14]
When I right click to delete a note, it asks to control finder, and when I say "No" the app crashes

**Comment by rajatkulkarni95** [comment]
Yeah it needs access to that otherwise it would crash. If you don't wish to give Finder Access, can you set this setting as `Delete Permanently`?
<img width="905" alt="Screenshot 2024-11-25 at 09 10 59" src="https://github.com/user-attachments/assets/8552e515-2522-4057-9c78-4d6469f4b30a">

I'll add a patch to default to this so that it doesn't crash if you don't provide access

**Comment by rajatkulkarni95** [comment]
This has been fixed in 0.19.1
